src/Sparrow.Server/ByteString.cs

```diff
@@ -446,7 +446,7 @@ public ReadOnlySpan<byte> ToReadOnlySpan()
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public Span<byte> ToSpan()
+        public readonly Span<byte> ToSpan()
         {
             return new Span<byte>(Ptr, Length);
```

Comment:
@ayende can you please check the warnings here?

Reply:
yes, reverted.